src/arctic3d/modules/pdb.py

```diff
@@ -714,14 +658,14 @@ def preprocess_pdb(pdb_fname, chain_id):
     tidy_pdb_f : Path
         preprocessed pdb file
     """
-    model_pdb_f = selmodel_pdb(pdb_fname)
-    atoms_pdb_f = keep_atoms(model_pdb_f)
-    chained_pdb_f = selchain_pdb(atoms_pdb_f, chain_id)
-    occ_pdb_f = occ_pdb(chained_pdb_f)
+    # model_pdb_f = selmodel_pdb(pdb_fname)
+    # atoms_pdb_f = keep_atoms(model_pdb_f)
+    # chained_pdb_f = selchain_pdb(atoms_pdb_f, chain_id)
```

Comment:
awesome, i'm always +1 of removing pdb-tools from inside the code, but here you should also remove the comments 🤓

Reply:
good point, i was thinking about keeping them commented, but yes, better to remove them